google/cloud/pubsub/internal/tracing_message_batch_test.cc

```diff
@@ -28,6 +28,7 @@ namespace pubsub_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 using ::google::cloud::internal::MakeSpan;
+using ::google::cloud::internal::ToString;
```

Comment:
Is there such a thing?

Reply:
Yes. It is the ToString for the otel types defined in opentelemetry.cc.

Reply:
The namespace is maybe questionable.